### DISCUSSION TITLE
Left Navigation

### DISCUSSION THREAD

**Discussion by janstadt** [tabler/tabler#2013] [General]
Im curious if there is a component or example of how to build the left navigation that is present on the documentation page: https://tabler.io/docs/getting-started I like how it looks and would like to implement it with native bootstrap/tabler classes and was curious if there is an example outside of just looking at the code for the docs in the repo or in dev tools.

**Comment by ethancrawford** [comment]
@janstadt - you may have already seen it, but is the [sidebar layout](https://tabler.io/docs/ui/layout/page-layouts#sidebar-layout) live demo close to what you need?

**Reply by janstadt** [reply]
Thanks. I dont know if that is a 1:1 example though unfortunately since the sidebar layout is at the top level (replacing the top nav), whereas the getting started docs seems to be one level down (aka the top nav isnt overlapped and doesnt share the same context/linking). Getting Started looks like a custom layout based on the dom. Would love to see an example of getting started using tabler or provide a similar layout example that represents the getting started page better.
